code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -1021,7 +1092,6 @@
 	special_names = list("Stalin","Lenin","Trotsky","Marx","Comrade") //comrade comrade
 	species_traits = list(NOBLOOD,NO_UNDERWEAR,NOEYES)
 	fixed_mut_color = null
-	inherent_traits = list(TRAIT_NOBREATH, TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOGUNS,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER)

```

Comment:
what's this about?

Reply:
Am I misinterpreting things, or did you just reset soviet golems to having the default golem traits?

Reply:
yeah but there was no differences except for resist heat being removed for some reason so it was redundant

Reply:
although may readd it once I remove the NO_GUNS THING